pyproject.toml

```
@@ -57,4 +57,4 @@ exclude = [
 type = "github"
 
 [tool.semantic_release.branches]
-sandbox = { match = "sandbox", prerelease = true, prerelease_token = "wip",  changelog_file = "CHANGELOG.md",  commit = false }
\ No newline at end of file
+release = { match = "release.*", prerelease = true, prerelease_token = "wip",  changelog_file = "CHANGELOG.md",  commit = false }
```

```suggestion
release = { match = "release/*", prerelease = true, prerelease_token = "wip",  changelog_file = "CHANGELOG.md",  commit = false }
```

Please :)

Or whatever the equivalent would be to match branches named:

```
release/v1
release/also-this
release/etc
```